src/source_sampling.cpp

```diff
@@ -554,17 +588,16 @@ pyne::SourceParticle::SourceParticle() {
     z = -1.0;
     e = -1.0;
     w = -1.0;
-    c = -1;
 }
 
 pyne::SourceParticle::SourceParticle(double _x, double _y, double _z,
-                                     double _e, double _w, int _c) {
+                                     double _e, double _w, std::vector<int> _cell_list) {
```

Comment:
Initializers are typically preferred I think
```suggestion
pyne::SourceParticle::SourceParticle(double _x, double _y, double _z,
                                     double _e, double _w, int _c) {
                                     double _e, double _w, std::vector<int> _cell_list) : 
                                     x(_x), y(_y), z(_z), 
                                     e(_e), w(_w), c(_c), 
                                     cell_list(_cell_list) { }
```